AssetRipper.Library/Exporters/Audio/AudioEffectsDefinitions.cs

```diff
@@ -0,0 +1,84 @@
+namespace AssetRipper.Library.Exporters.Audio
```

Comment:
Put the AudioMixer stuff in its own namespace: `AssetRipper.Library.Exporters.AudioMixers`. It is very different from normal audio stuff.